src/components/Content/TaskManager/TaskForm/TaskForm.css

```diff
@@ -38,8 +42,7 @@
   color: #efefef;
   border-radius: 0 4px 4px 0;
   border-color: #a5a5a5;
-  margin-top: 1px;
-  margin-bottom: -1px;
+  height: 32px;
```

Comment:
@Moe-Shoman Maybe these 2 removed lines caused the form field to be a little too tall?